libs/data-display/src/DataTable.tsx

```diff
@@ -1081,6 +1084,7 @@ function DataTableColumn<T extends object>(_: DataTableColumnProps<T>) {
 
 DataTableColumn.defaultProps = {
   type: "DataTableColumn",
+  canSort: false,
```

Comment:
I'm sceptical what to place here as default. In one case you can say that sorting on the tables should be a way to go always, but also there are many cases where it's completely unnecessary.

Reply:
If the prop is named canSort, then I think it's logical that default value should be false, so if you want to enable sorting, you set it to true. If sorting were enabled by default, then the prop should be named something like disableSort and than you call that prop if you want to disable it for specific column. That's how I see it. 

So, implementation should be:`<DataTable.Column ... canSort />` for the first case (when canSort=false by default) when we want to enable sorting for column. 
If we want sorting to be enabled by default than I think it is better aproach to use  `<DataTable.Column ... disableSort />` than `<DataTable.Column ... canSort={false} />` for columns where we want to disable sorting.

Reply:
Yes, it seems reasonable. I'll merge this but we can additionally check whether to add `defaultCanSort`